implementations/rust/ockam/ockam_node/src/storage/database/migrations/20240212100000_split_policies.sql

```diff
@@ -0,0 +1,15 @@
+-- Rename `policy` table
+ALTER TABLE policy RENAME TO resource_policy;
+ALTER TABLE resource_policy ADD COLUMN resource_type TEXT NULL;
+DROP INDEX IF EXISTS policy_index;
+CREATE UNIQUE INDEX resource_policy_index ON resource_policy (node_name, resource, action);
+
+-- Create a new table for resource type policies
+CREATE TABLE resource_type_policy
+(
+    resource   TEXT NOT NULL, -- resource type
```

Comment:
Can we rename it to `type` or `resource_type`?

Reply:
Sure, I'll do that